### PR TITLE
Added option to give chars to preserve

### PR DESCRIPTION
**Pull request by DMOEdetc**
Now you can submit a string which will be a reference for chars to be preserved. To allow custom replacements without interference with the preserve string the replacement with the custom replacements is made after the regular replacement

### CODE REVIEW

**Comment by DMOEdetc** [comment]
Oh, and I added the uppercase ẞ to German dictionary

**Comment by sindresorhus** [comment]
Please run `npm test` locally and ensure it passes.

**Comment by sindresorhus** [comment]
What's the specific use-case for this? I'm not against it, but no one else has requested this yet, so I'm curious how much of a need there really is for it.

File: replacements.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const replacements = [
 	// German umlauts
 	['ß', 'ss'],
+	['ẞ', 'Ss'],
```
Comment (sindresorhus) [inline replacements.js:4]: This is unrelated and should be a separate pull request.

File: index.js [hunk 2]
```diff
@@ -18,15 +21,15 @@ export default function transliterate(string, options) {
 
 	options = {
 		customReplacements: [],
+		preserveChars: '',
```
Comment (sindresorhus) [inline index.js:24]: Should be called `preservedCharacters` and accept an array of characters.

File: replacements.js (hunk 1 above)
Comment (DMOEdetc) [reply replacements.js]: sry, created an extra pr for this.
